docs/modules/ROOT/pages/includes/attributes.adoc

```diff
@@ -1,4 +1,4 @@
-:quarkus-version: 1.13.3.Final
+:quarkus-version: 1.13.4.Final
```

Comment:
Seems unrelated? We're not releasing here.

Reply:
Yep, 'mvn  clean package' changed the value, and I didn't cleaned it.